app/src/main/kotlin/Main.kt

```diff
@@ -1,8 +1,7 @@
 package me.john_
```

Comment:
should we change the namespace for this to something like `com.mctreesearch4j.xyz`

Reply:
No, this is the app, which won't be shipped. It doesn't matter what package it is.